### ISSUE TITLE
Trying to train transformer like the NMT Keras Transformer, for passage summarization task. The model's masked accuracy goes upto ~30% but the transformer's output is very poor. Could you help me understand what I might be doing wrong?

### ISSUE THREAD

**Issue by thewinterknight14** [tensorflow/tensorflow#58549]
<details><summary>Click to expand!</summary> 
 
 ### Issue Type

Performance

### Source

source

### Tensorflow Version

tf 2.8

### Custom Code

Yes

### Current Behaviour?

```shell
NMT task on the keras transformer as showed in the tutorials (https://www.tensorflow.org/text/tutorials/transformer) performs well. But I'm unable to get decent performance on a passage summarization task on the `cnn_dailymail` dataset. I can't understand what I might be doing wrong.
```

### Standalone code to reproduce the issue

```shell
https://colab.research.google.com/drive/12sbAGKESj8qynO-pb1K6gh-CZIvCxzfo#scrollTo=TlOaOjwwWN4K
```

### Relevant log output

_No response_</details>

**Comment by SuryanarayanaY** [comment]
Hi @thewinterknight14,
Iam unable to access the provided colab file. I request you to please convert it into Github gist and share the same with us.

Thankyou!

**Comment by thewinterknight14** [comment]
https://github.com/thewinterknight14/frost_queen/blob/master/transformer_passage_summarization.ipynb

**Comment by SuryanarayanaY** [comment]
Hi @thewinterknight14 ,

As per the gist you attached i understand that you need to improve the performance of your model right ?. For such issues we have dedicated [TF Forum](https://discuss.tensorflow.org/) where you can post the issues and ask for suggestions from larger community.Iam confident you may get some help there.

Thankyou!
